HISTORY.md

```diff
@@ -1,3 +1,8 @@
+unreleased
+===================
+
+  * deps: http-errors@~1.7.3
```

Comment:
We should add a sub bullet here with the inherits change that is part of this bump since it will be a visible change when installing send.

Reply:
Makes sense! I fixed at d34bc21